File: 5.py
# Modify the previous program such that only multiples of three or five are considered in the sum eg 3,5,6,9,10,12,15 for n = 17

n = input("Enter a number: ")

sum = 0

for i in range(1, n+1):
	if ((i % 3)==0) or ((i % 5)==0):
		print i
		sum = sum + i

print("The sum of 1 to %d, was %d" % (n, sum))

File: simple-exercises/5.c
#include <stdio.h>

int main()
{
    int n;
    int sum = 0;

    printf("Please enter a number: ");

    scanf("%d", &n);

    for(int i = 1; i<=n; i++)
    {
        if (((i%3)==0) || ((i%5)==0))
        {
            sum += i;
        }
        
    }

    printf("The sum of 1 to %d, is %d", n, sum);


    return 0;
}